### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by ghost** [airbnb/Showkase#298] [General]
Hi there! I've been looking into combining Showkase with Paparazzi recently, and I stumbled upon this PR (https://github.com/airbnb/Showkase/pull/294) which does exactly what I needed! It was only merged last week, so obviously the latest release doesn't include the new `showkase-screenshot-testing-paparazzi` artifact yet. I'm super excited to try it out - could anyone give me any indication of when you expect to make the next release? Thanks!

**Comment by vinaygaba** [comment]
I plan to use it myself as well, just need to make some minor changes first. I expect a new release by early next week.

**Reply by ghost** [reply]
Hi, I see there's no new release yet - are we waiting for some of the open PRs to merge? No rush, just to know for my own planning. Cheers!

**Reply by vinaygaba** [reply]
@DPGLuke Yes, the problem is that it's not working as I'd like it to at scale due to an issue in either Paparazzi or Layoutlib. Here's more details - https://github.com/cashapp/paparazzi/issues/630#issuecomment-1440679837

**Reply by ghost** [reply]
Thanks for the update!
